runelite-client/src/main/java/net/runelite/client/plugins/wiki/WikiPlugin.java

```diff
@@ -258,6 +261,17 @@ private void onMenuOptionClicked(MenuOptionClicked ev)
 					{
 						LinkBrowser.browse(WIKI_BASE + "/w/" + URLEncoder.encode(skillRegex.group(1)) + "?" + UTM_PARAMS);
 					}
+				case MENUOP_REWARDS:
+					ev.consume();
+					String questNA = Text.removeTags(ev.getMenuTarget());
+					LinkBrowser.browse(WIKI_BASE + "/w/" + URLEncoder.encode(questNA.replace(' ', '_')) + quickguide + "?" + UTM_PARAMS + "#Rewards");
+					break;
+
+                case MENUOP_WELCOMETOMYGUIDE:
+                    ev.consume();
+                    String questSM = Text.removeTags(ev.getMenuTarget());
+                    LinkBrowser.browse("https://www.youtube.com/results?search_query=Slayermusiq1+" + URLEncoder.encode(questSM.replace(' ', '+')) );
```

Comment:
We probably don't want to be linking to arbitrary youtubers

Reply:
I wouldn't call slayermusiq arbitrary, if we linked to *any* video guides it would make sense to use his. Still not a fan of the idea though

Reply:
Soup or bust